catalog/core/catalog-core-api/src/main/java/ddf/catalog/event/DeliveryMethod.java

```diff
@@ -59,6 +59,8 @@
      *
      * @param metacard the {@link Metacard} that was deleted
      */
-    public boolean ping();
+    public default boolean isAvailable() {
```

Comment:
Adding default methods to the core API will break backwards compatibility since integrators will need to recompile with this version of the API to work in the container.